profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/CrashReportingWindows.cpp

```diff
@@ -225,23 +239,176 @@ std::vector<ModuleInfo> CrashReportingWindows::GetModules()
                     resolvedModuleName = moduleName;
                 }
 
-                modules.push_back({ (uintptr_t)moduleInfo.lpBaseOfDll, (uintptr_t)moduleInfo.lpBaseOfDll + moduleInfo.SizeOfImage, std::move(resolvedModuleName) });
+                ModuleInfo module{ (uintptr_t)moduleInfo.lpBaseOfDll, (uintptr_t)moduleInfo.lpBaseOfDll + moduleInfo.SizeOfImage, std::move(resolvedModuleName), false, 0, 0 };
+
+                FillPdbInfo((uintptr_t)moduleInfo.lpBaseOfDll, module);
+
+                modules.push_back(module);
```

Comment:
you can call `std::move` here

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/6176/commits/24a2f6ef5fa63761ad269b47abe6a5e638cf06d4